deps/build.jl

```diff
@@ -32,7 +32,7 @@ function try_local_installation()
     # the path (directly the callable library or # the CPLEX executable) or from
     # an environment variable.
     cpxvers = [
-        "128", "1280", "129", "1290"
+        "128", "1280", "129", "1290", "1210", "12100"
```

Comment:
Hmm. So the need for `129` and `1290` is because there was a difference how CPLEX named their executables on Windows/NIX. I guess keeping both is fine, but it would be nice to know if they've settled on a single naming convention now that they're up to 4 letters.

Reply:
On Windows, the environment variable is called `CPLEX_STUDIO_DIR1210`, but the DLL is `cplex12100.dll`. I don't think there is any change with respect to the previous versions of CPLEX.

Reply:
Great! They kept it annoying then :)